j1939cat.c

```diff
@@ -121,7 +121,7 @@ static ssize_t j1939cat_send_one(struct j1939cat_priv *priv, int out_fd,
 		return -EINVAL;
 	}
 
-	if (num_sent > buf_size) /* Should never happen */ {
+	if ((size_t)num_sent > buf_size) /* Should never happen */ {
```

Comment:
Here is better to change type of buf_size to ssize_t. We use ssize_t by calling this function any way.